ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -2654,24 +2654,29 @@ patch_panel:
 
 ```yaml
 ptp:
-  mode: < mode >
+  mode: < boundary | transparent >
   forward_unicast: < true | false >
-  clock_identity: < clock-id >
+  clock_identity: < "clock-id in xx:xx:xx:xx:xx:xx format" >
   source:
-    ip: < source-ip>
-  priority1: < priority1 >
-  priority2: < priority2 >
-  ttl: < ttl >
-  domain: < integer >
+    ip: < source-ip >
+  priority1: < integer >
+  priority2: < integer >
+  ttl: < 1-254 >
+  domain: < 0-255 >
   message_type:
     general:
       dscp: < dscp-value >
     event:
-      dscp: < dscp-Value >
+      dscp: < dscp-value >
   monitor:
     threshold:
-      offset_from_master: < offset >
-      mean_path_delay: < delay >
+      offset_from_master: < integer >
+      mean_path_delay: < integer >
+      missing_message:
+        sync: < integer >
+        follow_up: < integer >
+        delay_resp: < integer >
+        announce: < integer >
```

Comment:
In the CLI each of these options must have 2-255 intervals and/or 2-255 sequence-ids
Suggestion
```
        missing_message:
          sync:
            intervals: < 2-255 >
            sequence_ids: < 2-255 >
        ...
```

Reply:
Updated